test/system/admin/notes_test.rb

```diff
@@ -6,18 +6,20 @@ class NotesTest < ApplicationSystemTestCase
   end
 
   test "adding a note to an item" do
-    @item = create(:item)
-    visit admin_item_url(@item)
+    swallow_js_errors(reason: "This intentionally causes a 422 error") do
```

Comment:
This is a really nice API to handle cases where we expect JS “errors” like 4xx responses.

Reply:
I updated the name as suggested. Good call, I like that better.